modules/gapi/test/streaming/gapi_streaming_tests.cpp

```diff
@@ -65,8 +64,6 @@ std::ostream& operator<< (std::ostream &os, const KernelPackage &e)
 }
 
 struct GAPI_Streaming: public ::testing::TestWithParam<KernelPackage> {
-    GAPI_Streaming() { initTestDataPath(); }
```

Comment:
Why is it copied into tests?

Reply:
Because i get segfault on my local pc.

Reply:
```
Assertion failed) testDataPath != nullptr in function 'initTestDataPath'
" thrown in the test fixture's constructor.
Segmentation fault (core dumped)
```

Reply:
Segfault is here because nobody catched exception.
Need to remove `GAPI_Assert(testDataPath != nullptr);` line from implementation and make this env variable optional (it make sense to left stdout message).

Reply:
Returned.